examples/adjoints/adjoints_ex5/adjoints_ex5.C

```diff
@@ -195,13 +187,20 @@ void set_system_parameters(HeatSystem & system,
   system.print_jacobian_norms    = param.print_jacobian_norms;
   system.print_jacobians         = param.print_jacobians;
   system.print_element_jacobians = param.print_element_jacobians;
-  system.print_element_residuals = param.print_element_residuals;
 
   // Solve this as a time-dependent or steady system
   if (param.transient)
     {
       UnsteadySolver *innersolver;
-      if (param.timesolver_core == "euler")
+      if (param.timesolver_core == "euler2")
+        {
+          Euler2Solver *euler2solver =
+            new Euler2Solver(system);
```

Comment:
I'm going to want to change this to avoid the raw heap pointers.

Reply:
But we were *already* using raw heap pointers for the old eulersolver, so I won't futz with it in this PR.